ansible/roles/containerd/vars/main.yml

```diff
@@ -0,0 +1,19 @@
+# containerd
+containerd_packages:
+  - containerd.io
+  - containernetworking-plugins
+
+containerd_dependencies:
+  - yum-utils
+  - device-mapper-persistent-data
+  - lvm2  
+
+docker_old_packages: 
+  - docker
+  - docker-client
+  - docker-client-latest
+  - docker-common
+  - docker-latest
+  - docker-latest-logrotate
+  - docker-logrotate
+  - docker-engine
```

Comment:
As mentioned before, are you sure that we need all these packages for containerd ?

Reply:
removed